ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-example-multinode-site.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: example-multinode-site-latest
+placementBindingDefaults:
+  name: example-multinode-site-placement-binding
+policyDefaults:
+  # categories: []
+  #controls:
+  #  - PR.DS-1 Data-at-rest
+  namespace: ztp-site
+  # Use an existing placement rule so that placement bindings can be consolidated
+  placement:
+    labelSelector:
+      # These policies will correspond to all clusters with this label:
+      site: "example-multinode"
+      # the below label (du-profile: "latest") might not necassery as the site policy applied to a single site.
+      du-profile: "latest"
+  remediationAction: inform
+  severity: low
+  # standards: []
+  namespaceSelector:
+    exclude:
+      - kube-*
+    include:
+      - '*'
+  evaluationInterval:
+    compliant: 10m
+    noncompliant: 10s
+policies:
+  # Create operators policies that will be installed in all clusters
+- name: example-multinode-site-config-policy-latest
```

Comment:
We have run into issues in the past with long policy names. Is this equivalent to the site policy name in the original?

Reply:
the acm policygen has the check for the policy name length (limit 63 char) no worry about that.